### PR TITLE
RAVTDBQ-246 - Group by Column Name Change In case of Literal with Alias

### PR DESCRIPTION
**Pull request by Shubham-Isalkar**
RAVTDBQ-246 - Group by Column Name Change In case of Literal with Alias

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java [hunk 1]
```diff
@@ -635,6 +634,18 @@ private SqlNode getGroupBySqlNode(Builder builder, int key) {
     } */
   }
 
+  private boolean aliasNotRequiredInGroupBy(Builder builder, int key) {
```
Comment (sauajmera) [inline core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java:637]: As this method returns a boolean value.
Could you please change name which start with either "is" or "does".
eg : isAliasNotRequiredInGroupBy

**Review by sauajmera — APPROVED** [review]

File: core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java (hunk 1 above)
Comment (Shubham-Isalkar) [reply core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java]: Done

File: core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java [hunk 2]
```diff
@@ -608,7 +608,7 @@ private List<SqlNode> generateGroupList(Builder builder,
 
   private SqlNode getGroupBySqlNode(Builder builder, int key) {
     boolean isGroupByAlias = dialect.getConformance().isGroupByAlias();
-    if (aliasNotRequiredInGroupBy(builder, key)) {
+    if (isAliasNotRequiredInGroupBy(builder, key)) {
```
Comment (sauajmera) [inline core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java:611]: changes looks good.

**Review by sauajmera — APPROVED** [review]

**Review by deepakkadam-dm — COMMENTED** [review]
Looks ok

**Pull request merged**